3rdParty/fuerte/include/fuerte/types.h

```diff
@@ -57,6 +57,8 @@ StatusCode constexpr StatusInternalError = 500;
 StatusCode constexpr StatusServiceUnavailable = 503;
 StatusCode constexpr StatusVersionNotSupported = 505;
 
+std::string status_code_to_string(StatusCode);
```

Comment:
I think for all fuerte types this kind of function is called `to_string`. And this allows ADL for this type using just `to_string`.

Reply:
I was squeamish about that name because StatusCode is
```
using StatusCode = std::uint32_t;
```,
and wanted to avoid ambiguity with `std::to_string`. But I don't have a very strong opinion on that and am willing to change it.